stencil/src/components/resource-links/resource-links.tsx

```diff
@@ -0,0 +1,68 @@
+import { Component, Prop, State, h } from "@stencil/core";
+// import { resources } from "./resources";
+
+// Scroll to top button.
+@Component({
+    tag: "pulumi-resource-links",
+    styleUrl: "resource-links.scss",
+    shadow: false,
+})
+export class ResourceLinks {
+    @Prop()
+    packageName: string;
+
+    @Prop()
+    moduleName: string;
+
+    @Prop()
+    resourceName: string;
+
+    @State()
+    relatedResources: {};
+
+    componentWillLoad() {
+        // Fetch JSON file containing resource link information.
+        fetch("https://www.pulumi.com/uploads/related-resources/2022-12-15.json")
+        .then(resp => resp.json())
+        .then((response) => {
+            this.relatedResources = response;
+        })
+        .catch(err => console.error(err))
+    }
+
+    render() {
```

Comment:
The logic here could arguably also be done in `componentWillLoad`. That way this logic doesn't run on every render. Also you wouldn't have to store the entire resource response in the state, you could just store the related resources (if there is one).